core/src/test/scala/unit/kafka/consumer/ZookeeperConsumerConnectorTest.scala

```diff
@@ -423,7 +423,7 @@ class ZookeeperConsumerConnectorTest extends KafkaServerTestHarness with Logging
       children.toSeq
     }
     childrenAsSeq.map(partition =>
-      (partition, zkUtils.zkClient.readData(path + "/" + partition).asInstanceOf[String]))
+      (partition, zkUtils.zkClient.readData(ZkUtils.getZkPath(path, partition)).asInstanceOf[String]))
```

Comment:
@SinghAsDev Could you please clarify? I just updated the path concatenation with the utility method call, as in other places in the PR.